.gitattributes

```diff
@@ -5,6 +5,7 @@
 /.gitignore export-ignore
 /.scrutinizer.yml export-ignore
 /.travis.yml export-ignore
+/bin export-ignore
```

Comment:
This way, the CLI utils will be only available during the development of the DBAL itself but not to other projects. See https://github.com/doctrine/dbal/pull/3956 for example.